### PR TITLE
feat: add opensea api key

### PR DESCRIPTION
**Pull request by alaibe**
Add opensea api key and config

### CODE REVIEW

File: services/wallet/service.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 )
 
 // NewService initializes service instance.
-func NewService(db *sql.DB, rpcClient *rpc.Client, accountFeed *event.Feed) *Service {
+func NewService(db *sql.DB, rpcClient *rpc.Client, accountFeed *event.Feed, openseaAPIKey string) *Service {
```
Comment (Samyoul) [inline services/wallet/service.go:16]: Is the key being introduced to the application at build time?

Comment (alaibe) [reply services/wallet/service.go:16]: In the case of desktop, the key is injected via jenkins at build: https://github.com/status-im/status-jenkins-lib/pull/38

Comment (Samyoul) [reply services/wallet/service.go:16]: Ok, injected into the client build. Thank you.

**Review by Samyoul — APPROVED** [review]

**Review by richard-ramos — APPROVED** [review]

File: services/wallet/service.go (hunk 1 above)
Comment (flexsurfer) [reply services/wallet/service.go:16]: @alaibe what if we inject it directly in status-go ? overwise we have to inject it in mobile and desktop and pass it in config

Comment (alaibe) [reply services/wallet/service.go:16]: We talk about this solution with @cammellos as an intermediate solution for mobile until mobile use loginWithConfig

Comment (cammellos) [reply services/wallet/service.go:16]: Yes, we should drive this config from the client (status-react), as desktop does, but we need to swap the endpoint, but that's a risky change for us and not hotfix material. 
For now, we can fallback on build-inject time key if the config is empty and then move to use `loginWithConfig`.
Though I doubt we will hotfix this, so we might as well just use `loginWithConfig` and release in 1.19

**Pull request merged**